helmfile.d/values/falco/falco-common.yaml.gotmpl

```
@@ -164,9 +164,18 @@ customRules:
           (container.image.repository = quay.io/calico/node and proc.name = calico-node) or
           (container.image.repository = registry.k8s.io/dns/k8s-dns-node-cache and proc.name = node-cache)
         )
+
+    {{- if and (eq .Values.calicoAccountant.enabled true) (eq .Values.calicoAccountant.backend "nftables") }}
```

```suggestion
    {{- if and .Values.calicoAccountant.enabled (eq .Values.calicoAccountant.backend "nftables") }}
```
Nit: Anything non empty and non false will be true in the world of go-templates :slightly_smiling_face:

---

@aarnq @OlleLarsson Ye I did not think about that, but I see we do the same if cases in other places in the same [file](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile.d/values/falco/falco-common.yaml.gotmpl#L16-L24), should I address these as well?

---

If it's quick to change, then yes please :)

---

So I changed it now, but something interesting is that it now seems to treat `"false"` as being `true`, and any other string it complains about it not beeing a boolean :thinking: :sweat_smile:

---

Okay :thinking:

---

That sounds like the way it should be :)

---

That it complains about strings not being booleans yes, but the `"false"` = `true` does not :sweat_smile: any go template experts that have encountered this?

---

This is not a problem with any of our config, but if I override any `enabled` (e.g. `falco.rulesFiles.incubating.enabled`) variable in the override config to `"false"` it treats it as `true` which is strange to me since it does not allow any other strings besides `"false"` and `"true"`

---

I tested this for other charts as well and the same thing happens. Maybe it is fine to leave it as is, once the schema validation is implemented it should warn whenever we configure a boolean as string so this should not be a problem